roles/cuda_toolkit_install/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+system_verified: false
+
+linux_version:
```

Comment:
- Investigate how other roles name this variable.
- Use a prefix if there is no consensus.
- No need to declare in defaults if there is no default value